server/models/users.js

```diff
@@ -27,7 +27,7 @@ const namespaceHelpers = require('../lib/namespace-helpers');
 
 const allowedKeys = new Set(['username', 'name', 'email', 'password', 'namespace', 'role']);
 const ownAccountAllowedKeys = new Set(['name', 'email', 'password']);
-const allowedKeysExternal = new Set(['username', 'namespace', 'role']);
+const allowedKeysExternal = new Set(['username', 'namespace', 'role', 'email', 'name']);
```

Comment:
Why is this? As far as I remember, email and name are taken from LDAP. Thus they shouldn't be allowed to be changed.

Reply:
That was an implementation change, we wanted to save email and name into BD. But you're right.